src/components/MyServiceList/Delete/index.tsx

```diff
@@ -4,13 +4,17 @@ import { useUser } from '../../../hooks/useUser';
 import { useRecoilState } from 'recoil';
 import { FixService } from '../../../Atom/Atoms';
 import useFetch from '../../../hooks/useFetch';
+import { ServiceCheckList } from '../../../Atom/Atoms';
+import { ClientListType } from '../../../types';
 
 export default function DeleteService() {
   const [user, getUser] = useUser(false);
   const [fix, setFix] = useRecoilState(FixService);
```

Comment:
```suggestion
  const setFix = useSetRecoilState(FixService);
```
recoil에서 setter 함수만 사용하고 싶다면 `useSetRecoilValue()`를 사용하면 됩니다

Reply:
[🎨 Update useRecoilState()를 useSetRecoilState()로 변경](https://github.com/GSM-MSG/GAuth-FrontEnd/commit/fae5d45b8630f6987703f4b1237e5473ec3635fd)